.github/workflows/ci.yml

```diff
@@ -0,0 +1,123 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request: ~
+
+jobs:
+
+  lint:
+    name: Lint
+    runs-on: 'ubuntu-latest'
+    timeout-minutes: 5
+
+    steps:
+      - name: 'Checkout'
+        uses: actions/checkout@v2
+
+      - name: 'Setup PHP'
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.0'
+
+      - name: 'Get composer cache directory'
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: 'Cache dependencies'
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+          restore-keys: ${{ runner.os }}-composer-
+
+      - name: 'Install dependencies'
+        run: composer update --prefer-dist
+
+      - name: 'Check style'
+        run: vendor/bin/php-cs-fixer fix --dry-run --no-interaction --diff
```

Comment:
I think it's better to put it in separate `yml` file and use https://github.com/marketplace/actions/oskar-php-cs-fixer to make the build faster.

Reply:
Thanks for the suggestion, but I prefer keeping it in sync with the dev workflow rather than using this image (not a strong argument, just a personal preference).
Keeping it in the same workflow is fine to me since it's still run in // and we don't need different triggers / scopes are relatively the same.